crates/gdtk-lint/src/lib.rs

```diff
@@ -1,14 +1,32 @@
-pub fn add(left: usize, right: usize) -> usize {
-    left + right
-}
+#![feature(let_chains)]
+
+pub mod lints;
+
+use gdtk_ast::Visitor;
+
+use crate::lints::{redundancy, style, syntax};
+
+pub fn run_builtin_lints(file: &gdtk_ast::ASTFile) -> Vec<miette::MietteDiagnostic> {
```

Comment:
We should think about if there's a way to hardcode lints less. ruff seems to do it manually like here, rustc.. not really sure, still haven't looked at any actual lint source.